src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,13 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: color 0.3s ease,
+      transform 0.3s ease;
```

Comment:
If you create a variable for time transition, maybe it is suitable to use it here?)

Reply:
Agree with variable, and `ease` default u can delete it